docs/en_US/Release.rst

```diff
@@ -5,6 +5,93 @@
 Change Log
 ==========
 
+Release 2.3 - 6/9/2021
+----------------------
+
+Major Updates
+^^^^^^^^^^^^^
+
+Neural Architecture Search
+""""""""""""""""""""""""""
+
+* Retiarii Framework (NNI NAS 2.0) Beta Release with new features:
+
+  * Support new high-level APIs: ``Repeat`` and ``Cell`` (#3481)
+  * Support pure-python execution engine (#3605)
+  * Support policy-based RL strategy (#3650)
+  * Support nested ModuleList (#3652)
+  * Updated documentation
+**Note**: there are more exciting Retiarii features planned in future releases, please refer to `Retiarii Roadmap <https://github.com/microsoft/nni/issues/3301>`__  for more information.
+
+* Add new NAS algorithm: Blockwise DNAS FBNet (#3532, thanks the external contributor @alibaba-yiwuyao) 
+
+Model Compression
+"""""""""""""""""
+
+* Support Auto Compression Framework (#3631)
+* Support slim pruner in Tensorflow (#3614)
+* Support LSQ quantizer (#3503, thanks the external contributor @chenbohua3)
+* Improve APIs for iterative pruners (#3507 #3688)
+
+Training service & Rest
+"""""""""""""""""""""""
+
+* Support 3rd-party training service (#3662 #3726)
+* Support setting prefix URL (#3625 #3674 #3672 #3643)
+* Improve NNI manager logging (#3624)
+* Remove tensorboard code on nnictl (#3613)
+
+Hyper-Parameter Optimization
+""""""""""""""""""""""""""""
+
+* Add new tuner: DNGO (#3479 #3707)
+* Add benchmark for tuners (#3644 #3720 #3689)
+
+WebUI
+"""""
+
+* Improve search parameters on trial detail page (#3651 #3723 #3715)
+* Make selected trials consistent after auto-refresh in detail table (#3597)
+* Add trial stdout button on local mode (#3653 #3690)
+
+Examples & Documentation
+""""""""""""""""""""""""
+
+* Convert all trial examples' from config v1 to config v2 (#3721 #3733 #3711 #3600)
+* Add new jupyter notebook examples (#3599 #3700)
+* Add pipeline badge in README (#3589)
```

Comment:
this is not document or example, but developer/code health management. can move it to others.